zerver/tests/test_message_move_stream.py

```diff
@@ -1102,7 +1102,7 @@ def test_move_message_to_stream_and_topic(self) -> None:
             "iago", "test move stream", "new stream", "test"
         )
 
-        with self.assert_database_query_count(51), self.assert_memcached_count(14):
+        with self.assert_database_query_count(53), self.assert_memcached_count(14):
```

Comment:
Can you clarify what the 2 new queries are here? I can imagine a world where we'd only need 1 extra.

Reply:
So for this particular test, we have `"send_notification_to_old_thread": "true",` . So two topic links are generated - one in old thread, and one in new thread -- 1 query each.

Reply:
OK -- and those are different topics, so the actual queries would be different. Seems OK.